### ISSUE TITLE
Beginner-friendly OpenMined Terminology

### ISSUE THREAD

**Issue by balapriyac** [OpenMined/writing#153]
- Terms associated with DL
- Gradient descent to Fwd prop, Backprop

**Comment by cescapa** [comment]
@jaintj95 hi Tejas, if you need an assistant or editor please lmk and share the Gdoc. I can help with de-jargonising.

**Comment by Metrix1010** [comment]
Hi @jaintj95 can you provide a status update on this issue?

**Comment by emharsha1812** [comment]
Please assign me this issue. I want to work on this.

**Comment by emharsha1812** [comment]
https://blog.openmined.org/beginner-friendly-openmined-terminology/

Blog is Published!
